src/Tribe/Tracker.php

```diff
@@ -33,6 +33,9 @@ public function hook() {
 		// Track the Meta Updates for Meta That came from the correct Post Types
 		add_filter( 'update_post_metadata', array( $this, 'filter_watch_updated_meta' ), PHP_INT_MAX - 1, 5 );
 
+		// Before a meta is removed we mark that is has been modified
+		add_filter( 'delete_post_metadata', array( $this, 'filter_watch_updated_meta' ), PHP_INT_MAX - 1, 5 );
```

Comment:
That action has a completely different parameter list, we have to write two methods when we can use only one.